src/main/java/com/rarchives/ripme/utils/RipUtils.java

```diff
@@ -149,122 +149,156 @@ public static Pattern getURLRegex() {
     }
 
     public static String urlFromDirectoryName(String dir) {
-        String url = null;
-        if (url == null) url = urlFromImgurDirectoryName(dir);
-        if (url == null) url = urlFromImagefapDirectoryName(dir);
-        if (url == null) url = urlFromDeviantartDirectoryName(dir);
-        if (url == null) url = urlFromRedditDirectoryName(dir);
-        if (url == null) url = urlFromSiteDirectoryName(dir, "bfcakes",     "http://www.bcfakes.com/celebritylist/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "butttoucher", "http://butttoucher.com/users/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "cheeby",      "http://cheeby.com/u/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "datwin",      "http://datw.in/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "drawcrowd",   "http://drawcrowd.com/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir.replace("-", "/"), "ehentai", "http://g.e-hentai.org/g/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "fapproved", "http://fapproved.com/users/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "vinebox", "http://finebox.co/u/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "imgbox", "http://imgbox.com/g/", "");
-        if (url == null) url = urlFromSiteDirectoryName(dir, "modelmayhem", "http://www.modelmayhem.com/", "");
+        String url = urlFromImgurDirectoryName(dir);
+
+        if (url == null)
+            url = urlFromImagefapDirectoryName(dir);
```

Comment:
So this whole thing, as a style guideline one of three things can happen. `if` statement with a single statement body all on a single line (as it was before), the same with braces around the body (my preference), or indented WITH braces added around the body.

Absolutely please do not use the indented form without adding brackets.

IMO, the single-line forms are more readable in this case.

If you want to make a change here, please add brackets around the if bodies and it's up to you whether to reformat so the bodies are all on their own lines.

Also looking at this code now, I feel like there's got to be a better way of writing this bit of code (more efficient, etc.). If you make that sort of change, please do it is in a separate commit or pull request.